src/ReleaseHistory.md

```diff
@@ -13,7 +13,9 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
-## **v4.0.0-rc3** UNRELEASED
+## **v4.0.0-rc4** UNRELEASED
```

Comment:
@shaopeng-gh Why add `rc4` here? We should just tag `rc3` as `Unreleased`.

Reply:
people will be adding to the last one, since rc3 is freezed, adding a new one rc4 above to avoid that.